website/docs/compute/cloud-deployment/configure-storage.md

```diff
@@ -0,0 +1,92 @@
+# Configure Storage
+
+By default, Compute will store compiled functions and function source code on the local storage in your Kubernetes cluster. For greater scalability, Compute can be configured to store artifacts in cloud-based object storage like Amazon S3 or Google Cloud Storage.
+
+To configure a storage bucket, provide the `SCC_STORAGE_PATH` environment variable to both the control plane and builder, e.g. `s3://my-bucket` for Amazon S3 or `gs://my-bucket` for Google Cloud Storage. For Kubernetes deployments, this is done in `.suborbital/scc-controlplane-deployment.yaml` under the `controlplane` and `builder` containers sections, and for local docker-compose deployments, this is done in `docker-compose.yaml` under the `scc-control-plane` and `scc-builder` services.
+
+## Authentication
+
+Bucket authentication varies between cloud providers.
+
+### Amazon S3
+
+You will need to also supply the `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, `AWS_SESSION_TOKEN` and `AWS_REGION` environment variables to both the control plane and the builder. See the [AWS authentication documentation](https://aws.github.io/aws-sdk-go-v2/docs/configuring-sdk/) for details. It is also possible to store the configuration as a Kubernetes secret, similar to the Google Cloud Storage configuration.
```

Comment:
note: this link only covers sdk v2, by default gocloud(and thus scn) uses sdk v1 which has it's own credential sourcing strategy. 

The env vars work for both but I would just go ahed and include both. ( see the notion doc which explains how to enable sdk2 and has links for both)